src/fromager/wheels.py

```diff
@@ -102,6 +105,9 @@ def build_wheel(
         sdist_root_dir=sdist_root_dir,
         version=version,
     )
+    # End the timer
+    end = datetime.now().replace(microsecond=0)
+    logger.info(f"{req.name}: time taken to build wheel {end - start}")
```

Comment:
Does this output go to stdout?

What happens to the `fromager step build` command output with this info log message here? The command is supposed to print the filename it creates in a way that a shell script can grab it and do something with it. Will that still work with this change here?

Reply:
Yeah, I can see it on terminal and in file `*-prepare-build.log` file
I also ran `test_build_steps.sh` to see if it works.

Is there anything else / another way that I need to test?

Reply:
What happens if you try to capture the output of a build command? Something like `wheel=$(fromager step build ...)`. Do you get just the filename, or does it include this new log message, too?

Reply:
I get just the filename
I ran the following command
```
OUTPUT="$(fromager \
        --log-file "$OUTDIR/build-logs/${dist}-prepare-build.log" \
        --work-dir "$OUTDIR/work-dir" \
        --sdists-repo "$OUTDIR/sdists-repo" \
        --wheels-repo "$OUTDIR/wheels-repo" \
        --wheel-server-url "${WHEEL_SERVER_URL}" \
        step build-wheel "$dist" "$version")"

echo "${OUTPUT}"
```
I get output as

`++ echo /home/rdevasth/fromager/e2e-output/wheels-repo/build/stevedore-5.2.0-py3-none-any.whl`

Reply:
Good, that's what I was hoping.